### ISSUE TITLE
IOS error "no valid caps-environment entitlement string found for application"

### ISSUE THREAD

**Issue by sibisandaniel86** [hansemannn/titanium-firebase-cloud-messaging#157]
Is this something to to with the module or firebase configuration ?

[ERROR] {
[ERROR] code = 3000;
[ERROR] error = "no valid \U201caps-environment\U201d entitlement string found for application";
[ERROR] source = {
[ERROR] setWindow = {
[ERROR] };

**Comment by m1ga** [comment]
did you enable push in your profile that you've created in the Apple developer page?

**Comment by sibisandaniel86** [comment]
yes , "Apple Push Notifications service (APNs)" is enabled in the key used in firebase conf.

**Comment by m1ga** [comment]
try https://stackoverflow.com/a/28114718/5193915 or https://stackoverflow.com/a/27324311/5193915
